pkg/arvo/app/graph-store.hoon

```diff
@@ -311,6 +313,31 @@
         (add-node-list resource graph mark (sort-nodes nodes))
       ==
       ::
+      ++  check-for-duplicates
+        |=  [=graph:store nodes=(set index:store)]
+        ^-  ?
+        =/  node-list  ~(tap in nodes)
+        |-
+        ?~  node-list  %.n
+        ?:  (has-node graph i.node-list)  %.y
+        $(node-list t.node-list)
```

Comment:
imo it's nicer to just do `?|` or w/e instead of explicitly returning booleans, when doing boolean logic.